app/app.go

```diff
@@ -565,11 +571,13 @@ func NewExocoreApp(
 		app.AssetsKeeper,     // assets for vote power
 	)
 
-	(&app.OperatorKeeper).SetHooks(
+	app.DelegationKeeper.SetHooks(app.StakingKeeper.DelegationHooks())
+
+	app.OperatorKeeper.SetHooks(
 		app.StakingKeeper.OperatorHooks(),
 	)
 
-	(&app.EpochsKeeper).SetHooks(
+	app.EpochsKeeper.SetHooks(
```

Comment:
When fixing conflicts use
```go
epochstypes.NewMultiEpochHooks(
			app.StakingKeeper.EpochsHooks(), // at this point, the order is irrelevant.
			app.ExomintKeeper.EpochsHooks(), // however, this may change once we have distribution
		),
```